velox/exec/tests/RowContainerTest.cpp

```diff
@@ -466,8 +466,42 @@ class RowContainerTest : public exec::test::RowContainerTestBase {
     }
   }
 
+  template <typename T>
+  void testRowContainerEqualAPIs(const TypePtr& type, const VectorPtr& values) {
+    auto rowContainer = makeRowContainer({type}, {type}, false);
+    int numRows = values->size();
+    DecodedVector decodedWithNulls(*values);
+    auto rows = storeRows(decodedWithNulls, numRows, *rowContainer);
```

Comment:
It looks like the test loops are very similar. Can you abstract a common function for it.